JDLApplicationOptions.md

```diff
@@ -1,38 +1,38 @@
-| uaaBaseName          | STRING   | Literal               | "uaa"                                                                                | Mandatory for gateway and microservices if auth type is uaa,  must be between double-quotes                   |
-| reactive             | BOOLEAN  | Boolean               | false                                                                                |                                                                                                              |
-| enableSwaggerCodegen | BOOLEAN  | Boolean               | false                                                                                |                                                                                                              |
-| skipServer           | BOOLEAN  | Boolean               | false                                                                                |                                                                                                              |
-| clientFramework      | STRING   | Literal               | angularX, react                                                                       |                                                                                                              |
-| enableHibernateCache | BOOLEAN  | Boolean               | true                                                                                 |                                                                                                              |
-| useSass              | BOOLEAN  | Boolean               |                                                                                      |                                                                                                              |
-| authenticationType   | STRING   | Literal               | jwt, uaa, session, oauth2                                                               | uaa for UAA apps,  jwt otherwise                                                                              |
+| clientPackageManager | STRING   | EnumLiteral           | npm, yarn                                                                             | One of the listed values or one of the prod database type                                                    |
+| databaseType         | STRING   | EnumLiteral           | sql, mongodb, cassandra, couchbase, no                                                   | One of the listed values or one of the prod database type                                                    |
+| uaaBaseName          | STRING   | EnumLiteral           | "uaa"                                                                                | Mandatory for gateway and microservices if auth type is uaa,  must be between double-quotes                   |
```

Comment:
Here the default value is indeed "uaa", but a user can set one

Reply:
had a look to application_options.js, is '**../uaa**' the right one?

Reply:
I don't get what you mean, but I think changing it to a Literal should work

Reply:
About the value, I think you're right